version_tracking/v0.4.0/README.md

```diff
@@ -0,0 +1,110 @@
+# v0.4.0
+
+- [Documentation of Changes](#documentation-of-changes)
+- [Test the Changes](#test-the-changes)
+- [Results Summary](#results-summary)  
+
+---
+
+## Documentation of Changes
+
+- PR [#60](https://github.com/mwvgroup/Pitt-Google-Broker/pull/60)
+- [rename-resources.md](rename-resources.md): accommodate generic survey
+- [schemas.md](schemas.md): create schema files for broker utils and BigQuery
+- [pypi.md](pypi.md): publish `pgb-broker-utils`
+- See also: new docs at [../../docs/source/broker](../../docs/source/broker)
+
+---
+
+## Test the Changes
+
+__Dashboards__ (linked to specific times of tests below):
+- [ZTF: Consumer test](https://console.cloud.google.com/monitoring/dashboards/builder/broker-instance-ztf-v040?project=ardent-cycling-243415&dashboardBuilderState=%257B%2522editModeEnabled%2522:false%257D&startTime=20210509T171043-04:00&endTime=20210509T171543-04:00) (old topic, alerts flood in)
+- [ZTF: everything but Consumer](https://console.cloud.google.com/monitoring/dashboards/builder/broker-instance-ztf-v040?project=ardent-cycling-243415&dashboardBuilderState=%257B%2522editModeEnabled%2522:false%257D&startTime=20210509T155021-04:00&endTime=20210509T164021-04:00) (consumer simulator)
+- [DECAT: everything but Consumer](https://console.cloud.google.com/monitoring/dashboards/builder/broker-instance-decat-v040?project=ardent-cycling-243415&dashboardBuilderState=%257B%2522editModeEnabled%2522:false%257D&startTime=20210509T172821-04:00&endTime=20210509T180521-04:00) (consumer simulator)
+
+See also [View and Access Resources](view-resources.md).
+
+
+Create a broker instance
+```bash
+# get the code
+git clone https://github.com/mwvgroup/Pitt-Google-Broker
+cd Pitt-Google-Broker
+git checkout v/0.4.0/tjr
+cd broker/setup_broker
+
+# create the instance
+survey="decat"
+# survey="ztf"
+testid="v040"
+teardown="False"
+./setup_broker.sh "$testid" "$teardown" "$survey"
+
+# name some things
+consumerVM="${survey}-consumer-${testid}"
+nconductVM="${survey}-night-conductor-${testid}"
+zone="us-central1-a"
+
+# upload credentials
+consumerDir="/home/broker/consumer"
+localDir="/Users/troyraen/Documents/PGB/repo"
+sudo gcloud compute scp "${localDir}/krb5.conf" "${consumerVM}:/etc/krb5.conf" --zone="$zone"
```

Comment:
Why do these three lines require `sudo`?

Reply:
I believe it was because the files are copied to directories that are not owned by the user (in this case, me). But actually, this command doesn't work for me anymore. I now have to copy the files to a directory I own (which does not require sudo) and then ssh in and move them.